src/include/wiredtiger.in

```diff
@@ -1897,6 +1907,25 @@ struct __wt_session {
 	int __F(timestamp_transaction)(WT_SESSION *session, const char *config);
 	/*! @} */
 
+	/*!
+	 * Set the commit timestamp on a transaction numerically.  Prefer this over @ref
+	 * timestamp_transaction when the string parsing done in that method becomes a bottleneck.
+	 *
+	 * The WT_SESSION.timestamp_transaction_numeric method can only be used at snapshot
+	 * isolation.
+	 *
+	 * @requires_transaction
+	 *
+	 * @param session the session handle
+	 * @param which the timestamp you wish to set. See @ref WT_TX_TXN_TYPE for available
+	 * options.
+	 * Durable timestamp should be set only after setting the commit timestamp.
```

Comment:
There are actually lots of rules around how timestamps can be set, calling out durable timestamps here doesn't make sense to me (see the `timestamp_transaction` wording for the long list). I'd suggest removing this and simply pointing to the `timestamp_transaction` wording, that is, treat this API as a fast-path for the slow API.

Reply:
Good point, thanks. Fixed in db28358